Gemfile

```diff
@@ -90,6 +90,8 @@ group :development, :test do
 
   # To avoid polling for changes on Windows MRI, it needs this opt-in gem
   gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+
+  gem 'bundler-audit'
```

Comment:
What is the benefit of this over GitHub's Dependency Bot PRs?

Reply:
@iHiD : I don't want to advocate for neither.
I can remove it if you prefer.

Reply:
@iHiD : there is a plan to integrate `bundler-audit` with `bundler`, so I removed it.
Source: https://rubycentral.org/news/stf-announce/

Reply:
> I can remove it if you prefer.

I hadn't formed an opinion. It was a genuine question.

> There is a plan to integrate bundler-audit with bundler, so I removed it.

Interesting. Thanks for the source.